Source/WebInspectorUI/UserInterface/Views/CookiePopover.js

```diff
@@ -224,6 +230,12 @@ WI.CookiePopover = class CookiePopover extends WI.Popover
 
         this._secureCheckboxElement = createInputRow("secure", WI.unlocalizedString("Secure"), "checkbox", data.secure).inputElement;
 
+        this._partitionedCheckboxElement = createInputRow("partitioned", WI.unlocalizedString("Partitioned"), "checkbox", data.partitioned).inputElement;
+        let partitionKeyInputRow = createInputRow("partitionKey", WI.UIString("Partition Key"), "text", data.partitionKey);
+        this._partitionKeyInputElement = partitionKeyInputRow.inputElement;
+        this._partitionKeyInputElement.readOnly = true;
+        partitionKeyInputRow.rowElement.hidden = !data.partitioned;
```

Comment:
NIT: we dont need to do this here since we're gonna do it in `togglePartitionKeyRow`